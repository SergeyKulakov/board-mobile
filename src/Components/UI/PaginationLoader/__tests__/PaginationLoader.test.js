import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { PaginationLoader as Component } from '../index'

describe('PaginationLoader', () => {
  const defaultProps = {
    visible: true,
    containerStyle: {},
    size: 15,
    color: '#fff',
  }

  it('render snapshots', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    const wrapper2 = shallow(<Component {...defaultProps} visible={false} />)

    expect([wrapper, wrapper2]).toMatchSnapshot()
  })
})
