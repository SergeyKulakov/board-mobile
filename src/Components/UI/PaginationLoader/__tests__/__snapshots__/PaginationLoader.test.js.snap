// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationLoader render snapshots 1`] = `
Array [
  <Styled(View)
    style={Object {}}
  >
    <ActivityIndicator
      animating={true}
      color="#fff"
      hidesWhenStopped={true}
      size={15}
    />
  </Styled(View)>,
  "",
]
`;
